docs/src/charm/tutorial/basic-operations.md

```diff
@@ -0,0 +1,167 @@
+# Basic operations with the charm
+
+This tutorial walks you through common management tasks for your {{ product }}
+cluster using the `k8s` charm. You'll learn how to scale your cluster, manage
+workers, and interact with it using `kubectl`.
+
+## What you will need
+
+- A running {{ product }} cluster deployed with the `k8s` charm
+- The Juju [client][Juju client]
+- [Kubectl] (installation instructions included below)
+
+## Scaling Your Cluster
+
+The `k8s` charm provides flexibility to scale your cluster as needed by adding
+or removing control plane nodes or worker nodes.
+
+To increase the control plane's capacity or ensure high availability, you can
+add more units to the `k8s` application:
+
+```
+juju add-unit k8s -n 2
```

Comment:
this adds 2 more units not makes the number of units = 2?

Reply:
Yes! Let me fix that